plugins-compat-tester/src/test/java/org/jenkins/tools/test/NonStandardTagHookTest.java

```diff
@@ -79,8 +79,6 @@ public void testCheckMethodWithMinimumVersion() {
     public void testActuallyPerformsTheCheckoutWithVersionGreaterThanMinimum() {
 
         try {
```

Comment:
Can just delete the `try`-`catch`; if the method throws an exception, that is consider a failure (and the test runner will display the stack trace, unlike the current poor idiom).

Reply:
I agree testing is a bit poor here and should need rewrite (you can almost break whatever you want no tests will fail).
Definitely not a good situation etc... but not sure such change should be a part of this PR as it's not really related.